continuous-integration/dev-sec-ops/penetration-testing/penetration_testing.md

```diff
@@ -0,0 +1,22 @@
+# Penetration Testing
+
+A penetration test is a simulated attack against your application to check for exploitable security issues.
+For further reading  see [CSEDevSecOps PenetrationTesting Scenarios](https://github.com/microsoft/CSEDevSecOps/tree/master/Scenarios/PenetrationTesting)

```

Comment:
Given this is an open source playbook that we share with our customers and those outside Microsoft I don't think we should include this link.  It is not accessible by people who are not part of the Microsoft org, it 404s.